githubteacher-orlando.md

```diff
@@ -0,0 +1,12 @@
+# Orlando, FL
+
+### Restaurants
+- The Pharmacy
```

Comment:
@mmeans01 the Pharmacy is a super secret hideout, it **is** a restaurant, I'll add more info to my file.